FastEasyMapping/Source/Core/Mapping/FEMMapping.m

```diff
@@ -53,6 +53,25 @@ - (instancetype)initWithEntityName:(NSString *)entityName rootPath:(NSString *)r
     return self;
 }
 
+#pragma mark - NSCopying
+
+- (instancetype)copyWithZone:(NSZone *)zone {
+    FEMMapping *mapping = [[self.class allocWithZone:zone] initWithEntityName:self.entityName];
```

Comment:
Also assignment of `objectClass` below needs to be removed in favor of using initializers